### ISSUE TITLE
Adobe Photoshop, Illustrator, Indesign support

### ISSUE THREAD

**Issue by adms-hu** [QL-Win/QuickLook#266]
**Is your feature request related to a problem? Please describe.**
Next to the missing quicklook feature from OSX there is something else. As a designer I miss so much to see the content of my Adobe Photoshop, Illustrator and Indesign files.

**Describe the solution you'd like**
I know your app wasn't meant to show nice thumbnails. Is there any chance to show psd, ai and ind files content in quicklook? The thumbnails should remain as Adobe provides it, but quicklook maybe can show the content.

**Comment by xupefei** [comment]
QL should already preview `.psd` and `.ai` files like normal images. Have you encountered any problem?

**Comment by crgriseus** [comment]
Yes you're right. :O I was confused, because I installed sagethumbs already, and it's showing the thumbs by default. I thought your app read from thumbnails only. Now I tested it out in the office. (No sagethumbs here) Some AI files shows code only, not the graphics.

One more thing. There is no real full screen, I can see the top bar when I press the full screen button. On OSX if you want to make a presentation from a bunch of jpg, png, video file, you just have to go full screen, and you have a powerful and easy access presentation tool.

**Comment by krasnovpro** [comment]
Confirm. QL can't see ai cc thumbs (which saved without pdf layer).
![190405120532](https://user-images.githubusercontent.com/886676/55616770-3dce7a00-579b-11e9-91d5-b1bb75f8b320.png)

**Comment by gaborvaraljai** [comment]
I have an .ai file that doesn't have a preview. Can you check it why doesn't work?

here is the link for the file:
https://1drv.ms/u/s!AiW3y1uYkNeiz-1F2FtMJeSTo6tphw?e=1EOlag
![ai_file_preview](https://user-images.githubusercontent.com/65733117/82595697-5a4b9280-9ba6-11ea-8835-e67c0815c1bf.png)

**Comment by schroef** [comment]
> Confirm. QL can't see ai cc thumbs (which saved without pdf layer).
> ![190405120532](https://user-images.githubusercontent.com/886676/55616770-3dce7a00-579b-11e9-91d5-b1bb75f8b320.p
Ng]
Osx can't neither, that's the limiting factor when saving without pdf setrings

**Comment by emako** [comment]
Any supporting about abobe files, just check https://github.com/QL-Win/QuickLook/issues/1516
